test/controllers/provisioning_templates_controller_test.rb

```diff
@@ -55,6 +55,11 @@ class ProvisioningTemplatesControllerTest < ActionController::TestCase
     assert_template 'new'
   end
 
+  test "export" do
+    get :export, { :id => templates(:pxekickstart).to_param }, set_session_user
+    assert_response :success
```

Comment:
Please also add the response assertions in the API tests to both of these UI controller tests.